backend/src/web_server.py

```diff
@@ -66,6 +66,7 @@ def shutdown(self):
         except socket.error as err:
             pass
 
+# Change to create thread per user (currently recieves commands out of order)
```

Comment:
Good idea! I'll work on that! 

that would probably work a lot better with scaling?